secret_settings.py

```diff
@@ -25,6 +25,15 @@
             "PORT": "",
         },
     },
+    # Gmail SMTP Relay
+    # https://support.google.com/a/answer/2956491?hl=en
+    "EMAIL": {
+        "HOST": "smtp-relay.gmail.com",
+        "PORT": 587,
+        "USERNAME": "noreply@cosmostue.nl",
+        "PASSWORD": "fakepassword",
+        "USE_TLS": True,
+    },
```

Comment:
@maximekabel @bcvandendool pls append this to the local `secrets.json` file. real password inside of the updated keepass db

Reply:
btw there are also ways of having multiple users, either via using the google apis, or via django-mailer, but that is sth that we can have a look at during mondays meeting

Reply:
I think we can rework the code for the future to include more users